lib/web-service/nginx-util.ts

```diff
@@ -88,6 +106,24 @@ function getSameSiteCookiesConfig(): string {
   return fs.readFileSync(resolvePath("nginx/samesite.conf"), "utf8");
 }
 
+/**
+ * Return the contents of an Nginx configuration file that patches
+ * `Set-Cookie` headers to use the `Partitioned` flag
+ *
+ * The output of this function is used with `createConfigMap` with `usePartionedCookiesLocations` provided.
+ */
+function getPartitionedCookiesConfig(locations: string[]): string {
+  const configs = locations.map(
+    (location) => `location ${location} {
+      proxy_cookie_path / "/; Partitioned";
+    }`,
+  );
+
+  return `server {
+    ${configs.join("\n")}
+  }`;
```

Comment:
I don't really know Nginx, but will such `server` block work?

I.e. assuming there is another `server` block (or maybe more, with different ports, hostnames, etc.), would Nginx enter this block too?

Reply:
comment: Looking at this, it looks like you've pointed out a flaw in my thought process:

https://docs.nginx.com/nginx/admin-guide/web-server/web-server/

This will probably never resolve the locations in the second block, the first will always be seen as the default.

I suppose the flag should probably change the default config, so it adds the locations into the default server block.

Reply:
> I suppose the flag should probably change the default config, so it adds the locations into the default server block.

I think that's the solution.